toolkit/scripts/daily_build.mk

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# Contains:
+#	- Rules to update manifests
+#	- Mechanism to update DAILY_BUILD_ID
+
+TOOLCHAIN_MANIFEST ?= $(TOOLCHAIN_MANIFESTS_DIR)/toolchain_$(build_arch).txt
+WORKER_CHROOT_MANIFEST = $(TOOLCHAIN_MANIFESTS_DIR)/pkggen_core_$(build_arch).txt
+
+lkg_workdir = $(BUILD_DIR)/daily_build_id
+
+######## CONFIGURE FLAGS ########
+
```

Comment:
```suggestion

# DAILY_BUILD_ID is the main point of entry. It can be one of 'lkg', 'lkg-force', or <build-id>. Based on this selection 'LKG_MANIFESTS' and 'FORCE_MANIFEST_UPDATES' are selected by default. They may be overwritten if desired.

# LKG_MANIFESTS will update the manifest files to match the commit in the LKG file. This does not make sense for arbitrary build-ids however, so only default to 'y' for 'lkg' or 'lkg-force'

# PKG_MANIFEST_OVERWRITE will ignore local changes to the manifests and replace them with the LKG version.

```